packages/notifi-dapp-example/src/components/Toggle.tsx

```diff
@@ -4,12 +4,14 @@ export type ToggleProps = Readonly<{
   disabled: boolean;
   checked: boolean;
   onChange?: () => void;
+  handleChange?: (val: boolean) => void;
 }>;
 
 export const Toggle: React.FC<ToggleProps> = ({
   disabled,
   checked,
   onChange,
+  handleChange,
```

Comment:
Do we need an additional handle change? 
Looks like we can simply utilize `onChange`